lib/galaxy/model/store/ro_crate_utils.py

```diff
@@ -220,7 +220,77 @@ def _add_workflows(self, crate: ROCrate):
 
             crate.license = self.workflow.license or ""
             crate.mainEntity["name"] = self.workflow.name
-            crate.mainEntity["subjectOf"] = cwl_wf
+
+            # Adding the creator information
+            if hasattr(self.workflow, 'creator_metadata') and self.workflow.creator_metadata:
+                creators = self.workflow.creator_metadata
+                if creators and isinstance(creators, list) and len(creators) > 0:
+                    first_creator = creators[0]
```

Comment:
Why only the first one ? Also this might be an organization and not a person.

Reply:
Oh did not think of that okay ! I'll will change that to have multiple people and the organisation